test/testcafe/spec/DeviceEnrollmentTerminalView_spec.js

```diff
@@ -87,7 +87,6 @@ test
 test
```

Comment:
Fix for unrelated A11y failures: https://github.com/okta/okta-signin-widget/pull/3096/commits/bc030f0f67788c5a9af95805a32641f369d99a62